### ISSUE TITLE
Import CSV from MySQL to MsSQL JDBC Error

### ISSUE THREAD

**Issue by Made-of-Clay** [dbeaver/dbeaver#6951]
#### System information: 
- Operating system (distribution) and version: Ubuntu 18.04.3 LTS - Gnome 3.82.2
- DBeaver version: 6.2.2
- Additional extensions: MS SQL Server > SQL Server version 7.0.0.jre8

#### Connection specification:
- Database name and version
    - Mircosoft SQL Server 12.00.5000
    - MySQL 5.6.33-0ubuntu0.14.04.1
- Driver name
    - Mircosoft JDBC Driver 7.0 for SQL Server 7.0.0.0
    - MySQL Connector Java mysql-connector-java-5.1.44
- Do you use tunnels or proxies (SSH, SOCKS, etc)? no?

#### Describe the problem you're observing:
I am exporting a user table from MySQL as CSV, changing column names, and importing to MsSQL. My import settings are all default. I get a JDBC error (see below)

#### Steps to reproduce, if exist:
See description above?

#### Include any warning/errors/backtraces from the logs
> org.jkiss.dbeaver.tools.transfer.stream.StreamTransferSession cannot be cast to org.jkiss.dbeaver.model.exec.jdbc.JDBCSession
org.jkiss.dbeaver.tools.transfer.stream.StreamTransferSession cannot be cast to org.jkiss.dbeaver.model.exec.jdbc.JDBCSession

**Comment by sigvehag** [comment]
I'm getting the same error message.

**System information:**
- Operating system (distribution) and version: Red Hat Enterprise Linux Server 7.7
- DBeaver version: 6.2.5
- Additional extensions: MS SQL Server


**Connection specification:**
- Database name and version
  - Unsure, but know it runs SQL Server 2016
- Driver name
  - Microsoft JDBC Driver for SQL Server

**Do you use tunnels or proxies (SSH, SOCKS, etc)?**
No

**Describe the problem you're observing:**
On importing a CSV-file to populate the tables in the the database I get an error.

**Steps to reproduce, if exist:**
I export the data from a SQL Server 2017 to CSV and try to import it into a SQL Server 2016. The databases have identical structure, and the database I'm trying to import the data into has no data.

**Include any warning/errors/backtraces from the logs**

```
Data transfer failed

Reason:
org.jkiss.dbeaver.tools.transfer.stream.StreamTransferSession cannot be cast to org.jkiss.dbeaver.model.exec.jdbc.JDBCSession
```

**Comment by serge-rider** [comment]
Fixed in #7385

**Comment by resquej** [comment]
Hi,

I found same issue but I am unable to see the solution.
Thank you

Task execution failed

Reason:
class org.jkiss.dbeaver.tools.transfer.stream.model.StreamTransferSession cannot be cast to class org.jkiss.dbeaver.model.exec.jdbc.JDBCSession (org.jkiss.dbeaver.tools.transfer.stream.model.StreamTransferSession is in unnamed module of loader org.eclipse.osgi.internal.loader.EquinoxClassLoader @78920de1; org.jkiss.dbeaver.model.exec.jdbc.JDBCSession is in unnamed module of loader org.eclipse.osgi.internal.loader.EquinoxClassLoader @2ef507f3)

**System information:**
Operating system (distribution) and version: Windows 10
DBeaver version: 6.3.4
**Connection specification:**

- Database name and version
Micrcosoft SQL Server 14.0.2027

**Driver name**
Microsoft JDBC Driver for SQL Server (MSSQL)
Do you use tunnels or proxies (SSH, SOCKS, etc)? no?
